Sources/Publish/Internal/PodcastFeedGenerator.swift

```diff
@@ -50,7 +50,7 @@ private extension PodcastFeedGenerator {
                   section: Section<Site>) throws -> PodcastFeed {
         try PodcastFeed(
             .unwrap(config.newFeedURL, Node.newFeedURL),
-            .title(config.title),
+            .title(config.title != nil ? config.title! : context.site.name),
```

Comment:
Perhaps use coalescing operator instead?
```
 .title(config.title ?? context.site.name),```

Reply:
Yeah, that'd be much better 👍